src/components/web3/ConnectButton.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import Image from 'next/image'
 import Link from 'next/link'
 import { FC, useState } from 'react'
-import { AiOutlineCheckCircle, AiOutlineDisconnect } from 'react-icons/ai'
+import { AiOutlineCheckCircle, AiOutlineDisconnect, AiOutlineMeh } from 'react-icons/ai'
```

Comment:
alternative to ignore evm addresses
```suggestion
import { AiOutlineCheckCircle, AiOutlineDisconnect } from 'react-icons/ai'
```